Modules/_ctypes/callproc.c

```diff
@@ -1893,8 +1899,21 @@ unpickle(PyObject *self, PyObject *args)
     return NULL;
 }
 
+/*[clinic input]
+_ctypes.POINTER as create_pointer_type
+
+    type as cls: object
+    /
+
+Creates and returns a new ctypes pointer type.
```

Comment:
I suggest using the imperative mode for anything that goes in the reference category of Diátaxis.

```suggestion
Create and return a new ctypes pointer type.
```